ployst/ui/views.py

```diff
@@ -14,6 +15,7 @@ def get_context_data(self, *args, **kwargs):
         providers = get_all_providers_meta()
         data.update({
             'providers': providers,
+            'core_url': settings.CORE_API_ADDRESS
```

Comment:
In particular when debugging why on Heroku the github provider hang up while talking to core. I wanted to make sure the port core was listening to was the one Heroku reported in the startup log.

I will remove this before merging. I'd rather later add debug toolbar.